templates/menu.html

```diff
@@ -3,4 +3,115 @@
 {% block content %}
 
 
+<h1 class="text-center">Nasze Menu</h1>
+
+<div class="background">
+    <div class="background-layer"></div> <!-- Pierwsza warstwa tła -->
+    <div class="background-layer"></div> <!-- Druga warstwa tła -->
+</div>
+
+
+<!-- Filter Buttons -->
+<div class="text-center my-3">
+    <button class="custom-btn filter-btn" data-filter="all">Wszystkie pozycje</button>
+    <button class="custom-btn filter-btn" data-filter="Dania Mięsne">Dania Mięsne</button>
+    <button class="custom-btn filter-btn" data-filter="Pizza">Pizza</button>
+    <button class="custom-btn filter-btn" data-filter="Sfiha">Sfiha</button>
+    <button class="custom-btn filter-btn" data-filter="Wege">Dania Wegetariańskie</button>
+    <button class="custom-btn filter-btn" data-filter="Ryba">Dania Rybne</button>
+    <button class="custom-btn filter-btn" data-filter="Sałatki">Sałatki</button>
+    <button class="custom-btn filter-btn" data-filter="Dodatki">Dodatki</button>
+    <button class="custom-btn filter-btn" data-filter="Desery">Desery</button>
+    <button class="custom-btn filter-btn" data-filter="Napoje">Napoje</button>
+</div>
+
+<!-- Menu Items -->
+<div class="row" id="menu-container">
+    {% for item in menu_items %}
+    <div class="col-md-4 menu-item" data-category="{{ item.category }}">
+        <img src="{{ url_for('static', filename='images/' + item.image) }}" alt="{{ item.name }}">
+        <h5>{{ item.name }}</h5>
+        <p class="price">{{ item.price }}</p>
+         <!-- Opis ukryty na początku -->
+        <div class="desc-container">
+            <p class="desc">{{ item.desc | safe }}</p>
+        </div>
+
+
+    </div>
+    {% endfor %}
+</div>
+
+<script src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
+<script>
+    $(document).ready(function() {
+        $(".filter-btn").click(function() {
+            var category = $(this).attr("data-filter");
+            if (category == "all") {
+                $(".menu-item").show();
+            } else {
+                $(".menu-item").hide();
+                $('.menu-item[data-category="' + category + '"]').show();
+            }
+        });
+    });
+</script>
+
+<!-- Skrypt do zmiany tła -->
+{% if background_script %}
+ <script>
```

Comment:
Don't duplicate your code. If you move it to a separate `.js` file you can include it in both HTML files